openresty/conf.d/nginx_lua.conf

```diff
@@ -0,0 +1,19 @@
+## http section configuration that is required for lua-resty-openidc
+# You may want to use your own resolver here. It's necessary for Lua to function.
+# If using a local DNS cache, you should use 127.0.0.1
+
+# resolver 8.8.8.8 8.8.4.4;
+# We have to use a local resolver to be able to resolve the DNS name of the
+# proxy servers configured in proxy_opts in server.lua
+resolver 127.0.0.1;
```

Comment:
I'm not sure if this is necessary? I think most of our systems already use localhost as the resolver.